tremor-pipeline/src/op/trickle/select.rs

```diff
@@ -564,8 +613,14 @@ fn get_or_create_group<'window>(
             });
             e.insert(k, v)
         }
+        RawEntryMut::Vacant(_) => {
```

Comment:
we probably should add an `error!` here as well to make sure this is not a silent error even if the error pipeline isn't set up.

Reply:
this is erroring out of the whole event which will log an error as tested here: https://github.com/tremor-rs/tremor-runtime/pull/828/files#diff-d1682b33030c689487d46d41c5f162f105858e9b3e5611918438a35295adc045R9

is this good enough?

Reply:
Ja! I didn't look up the chain :) sorry